src/sentry/static/sentry/app/views/settings/settingsIndex.jsx

```diff
@@ -96,48 +95,49 @@ class SettingsIndex extends React.Component {
               </Panel>
             </Box>
 
-            <Box w={1 / 3} px={2}>
-              {/* if admin */}
-              <Panel>
-                {!organization && <LoadingIndicator overlay />}
-                <HomePanelHeader>
-                  <HomeLinkIcon to={organizationSettingsUrl}>
-                    {organization ? (
-                      <AvatarContainer>
-                        <Avatar organization={organization} size={HOME_ICON_SIZE} />
-                      </AvatarContainer>
-                    ) : (
-                      <HomeIcon color="green">
-                        <InlineSvg src="icon-stack" size="44px" />
-                      </HomeIcon>
-                    )}
-                    <OrganizationName css={{lineHeight: '1.1em'}}>
-                      {organization ? organization.slug : t('Organization')}
-                    </OrganizationName>
-                  </HomeLinkIcon>
-                </HomePanelHeader>
-                <HomePanelBody>
-                  <h3>{t('Quick links')}:</h3>
-                  <ul>
-                    <li>
-                      <HomeLink to={`${organizationSettingsUrl}projects/`}>
-                        {t('Projects')}
-                      </HomeLink>
-                    </li>
-                    <li>
-                      <HomeLink to={`${organizationSettingsUrl}teams/`}>
-                        {t('Teams')}
-                      </HomeLink>
-                    </li>
-                    <li>
-                      <HomeLink to={`${organizationSettingsUrl}members/`}>
-                        {t('Members')}
-                      </HomeLink>
-                    </li>
-                  </ul>
-                </HomePanelBody>
-              </Panel>
-            </Box>
+            {organization && (
```

Comment:
I think we'll want to always display this section otherwise the following sections will jump around, and we want to avoid that.

We also attempt to come up with a best guess for the organization using `withLatestContext` - currently I believe it's the last used organization (via store) or just the first organization (most people only have 1 org)

Reply:
woops ^

Reply:
When there's no organization (ex: clicking through settings during the require 2FA flow), the page slows down and the spin wheel of death goes continuously. I got feedback that it seems broken. Do you want to keep this behavior, or could we display that there's no organization in another way?

Reply:
Ah, that makes sense, in this case, can we keep the "box" (maybe with a message saying "No Organization")?